lib/src/migrators/calc_interpolation.dart

```diff
@@ -37,10 +37,16 @@ class _CalculationInterpolationVisitor extends MigrationVisitor {
     const calcFunctions = ['calc', 'clamp', 'min', 'max'];
     final interpolation = RegExp(r'\#{\s*[^}]+\s*}');
     final hasOperation = RegExp(r'[-+*/]+');
+    final isVarFunc = RegExp(r'var\(\#{\s*[^}]+\s*}\)');
     if (calcFunctions.contains(node.name)) {
       for (var arg in node.arguments) {
         var newArg = arg.toString();
-        for (var match in interpolation.allMatches(arg.toString())) {
+        var varFuncArgs;
+        if (isVarFunc.hasMatch(newArg)) {
+          varFuncArgs = isVarFunc.allMatches(newArg);
+          newArg = newArg.replaceAll(isVarFunc, 'var()');
+        }
+        for (var match in interpolation.allMatches(newArg)) {
```

Comment:
If you're declaring an uninitialized variable, you need to give it an explicit type (in this case, it's `Iterable<RegExpMatch>?`).

Alternatively, you could just always set `varFuncArgs` to the result of calling `allMatches` and then check `varFuncArgs.isNotEmpty` instead of calling `hasMatch`.

Reply:
Thanks! Done